Feature/screens/mypage

duswo3o opened the pull request:
## #️⃣ Issue Number

<!--- ex) #이슈번호, #이슈번호 -->
close #15 

## 📝 요약(Summary)

<!--- 변경 사항 및 관련 이슈에 대해 간단하게 작성해주세요. 어떻게보다 무엇을 왜 수정했는지 설명해주세요. -->
- 로그인한 사용자의 경우 닉네임, 아이디, 팔로워, 팔로잉, 유료 서비스 정보, 캐릭터 관리 UI를 표시합니다
- 캐릭터가 없는 경우 "캐랙터 제작하기" 버튼이 나타납니다.


## 🛠️ PR 유형

어떤 변경 사항이 있나요?

- [x] 새로운 기능 추가
- [ ] 버그 수정
- [x] CSS 등 사용자 UI 디자인 변경
- [x] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [x] 코드 리팩토링
- [x] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [ ] 파일 혹은 폴더명 수정
- [ ] 파일 혹은 폴더 삭제

## 📸스크린샷 (선택)

![image](https://github.com/user-attachments/assets/4419c343-9e9b-42cb-b1fd-335f25d3b9a8)
![image](https://github.com/user-attachments/assets/e8855d6a-0198-4c7f-bf6a-3e626ee6926b)


## 💬 공유사항 to 리뷰어

<!--- 리뷰어가 중점적으로 봐줬으면 좋겠는 부분이 있으면 적어주세요. -->
<!--- 논의해야할 부분이 있다면 적어주세요.-->
<!--- ex) 메서드 XXX의 이름을 더 잘 짓고 싶은데 혹시 좋은 명칭이 있을까요? -->
- Apple 로그인 주석처리 하였습니다.

## ✅ PR Checklist

PR이 다음 요구 사항을 충족하는지 확인하세요.

- [x] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [x] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

duswo3o committed:
✨ Feat: implement mypage UI for logged-in users

로그이난 유저를 위한 마이페이지 화면
- 닉네임, 아이디, 팔로워, 팔로잉 정보 표시
- 프로필 공유 및 프로필 편집 버튼 추가
- 유료 서비스 관리 및 캐릭터 관리 UI 추가
- 캐릭터가 없을 경우 캐릭터 제작하기 버튼 표시

duswo3o committed:
🎨 Style: recfactor inline style to StyleSheet

인라인 스타일을 StyleSheet로 분리

duswo3o committed:
💄 Style: apple login annotation processing

apple 로그인 주석 처리

apr-5 reviewed (approved):
짱짱! 
마이 페이지에 있는 제타로 적혀 있는 부분들 베타로 수정하면 좋을 것 같아요!

duswo3o commented:
@apr-5 

넵! 수정하고 머지하겠습니다!

duswo3o committed:
🩹 Fix: change 'zeta' to 'beta' in mypage screen

마이페이지 화면 내 'zeta'로 표기된 부분을 'beta'로 수정
- 'zeta pro' -> 'beta pro'
- '광고 없이 zeta 하기' -> '광고 없이 beta 하기'

duswo3o commented:
beta로 수정하였습니다.

![image](https://github.com/user-attachments/assets/444dc2c7-e269-48df-a507-b4b2bb841cf6)

Drills-hub reviewed (approved):
좋아요~
